app_feup/lib/view/Widgets/GenericCard.dart

```diff
@@ -63,7 +63,9 @@ abstract class GenericCard extends StatefulWidget {
                 .textTheme
                 .display1));
       default:
-        return Container();
+        return contentChecker ? 
+        contentGenerator(content, context):
+        Container();
```

Comment:
![image](https://user-images.githubusercontent.com/26902818/65610849-d8b94b80-dfa9-11e9-8660-b48ac9d479bc.png)

This is not working for me. Shouldn't Container have an error message?